spec/visual_regression_tests/all_components_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+require "percy"
+require "uri"
+
+describe "visual regression test runner Percy", visual_regression: true do
+  it "takes a screenshot of each component" do
+    # Freeze time for consistency
+    travel_to(Time.zone.local(2020, 12, 1, 6, 0, 0))
```

Comment:
This stub won't be reset after the test which could lead to some surprising situations for any Ruby running after the test. You can fix this by running the code in a block or using a `travel_back` method at the end of the test

Reply:
Thanks for spotting this - fixed ⌛